src/uu/split/src/split.rs

```diff
@@ -47,6 +52,12 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
         "additional suffix to append to output file names",
         "SUFFIX",
     );
+    opts.optopt(
```

Comment:
if you are motivated, don't hesitate to port this to clap :) (in a different PR)

Reply:
in next PR